presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -666,12 +668,33 @@ output expressions:
 
     ORDER BY expression [ ASC | DESC ] [ NULLS { FIRST | LAST } ] [, ...]
 
-Each expression may be composed of output columns or it may be an ordinal
-number selecting an output column by position (starting at one). The
-``ORDER BY`` clause is evaluated after any ``GROUP BY`` or ``HAVING`` clause
+Each expression may be composed of output columns, or it may be an ordinal
+number selecting an output column by position, starting at one. The
+``ORDER BY`` clause is evaluated after any ``GROUP BY`` or ``HAVING`` clause,
 and before any ``OFFSET``, ``LIMIT`` or ``FETCH FIRST`` clause.
 The default null ordering is ``NULLS LAST``, regardless of the ordering direction.
 
+Note that, following the SQL specification, an ``ORDER BY`` clause only
+guarantees the order of rows for queries that immediately contain the clause.
```

Comment:
"guarantees" -- or "governs", "specifies"?

Reply:
changed to "affects"